warehouse/models/mart/gtfs/fct_hourly_rt_feed_files.sql

```diff
@@ -12,6 +12,8 @@ WITH
 int_gtfs_rt__daily_url_index AS (
     SELECT *
     FROM {{ ref('int_gtfs_rt__daily_url_index') }}
+    WHERE data_quality_pipeline IS TRUE
```

Comment:
nit: because this is a boolean column you can just say `WHERE data_quality_pipeline` (don't need the `IS TRUE`). It's probably fine to leave as-is given that `data_quality_pipeline` doesn't have the clearest name (like, it might be better if it were called `use_in_data_quality_pipeline`, but that's an upstream issue)